src/app/views/sap/create_mission.php

```diff
@@ -15,10 +15,10 @@
                 <?php
                 if (isset($errors) && count($errors) !== 0) {
                     foreach ($errors as $error) {
-                        echo "<p style=\"color:red\">$error</p>";
+                        echo "<p class=\"text-error\">$error</p>";
```

Comment:
This is another convention issue that I was going to bring up.

- The reason to use `"` in strings that get displayed to the user is that there is generally usage of variable insertion and `'` ASCII apostrophes in human readable text.
- The reason to use `'` in PHP code is that `"` come up in other kinds of code, such as HTML, and hence need to be escaped.

Wherever I've escaped the quotes, I've gone with picking the first one over the second one. I'll change it to whatever convention we agree upon, but please let it slide for now.